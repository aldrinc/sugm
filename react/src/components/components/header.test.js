import React from 'react';
import Header from './header';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

it('header renders correctly', () => {
    const tree = renderer.create(
      <Router>
<Header/>
      </Router>
      
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
