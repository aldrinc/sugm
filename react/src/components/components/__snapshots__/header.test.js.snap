// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders correctly 1`] = `
<header
  className="Apps__header"
>
  <div
    className="container-fluid"
  >
    <div
      className="row headder"
    >
      <div
        className="col-4 header_left"
      >
        <div
          className="nav-side-menu"
        >
          <div
            className="toggle-button"
            onClick={[Function]}
          >
            <svg
              className="svg-icon "
              height="100%"
              style={Object {}}
              viewBox="0 0 32 32"
              width={50}
              xmlns="http://www.w3.org/2000/svg"
              xmlnsXlink="http://www.w3.org/1999/xlink"
            >
              <g
                fill="{fill}"
                fill-rule="evenodd"
                id="Page-1"
                stroke="none"
                stroke-linecap="round"
                stroke-linejoin="round"
                strokeWidth="1"
              >
                <g
                  id="navigation-menu"
                  stroke="#000000"
                  transform="translate(1.000000, 1.000000)"
                >
                  <path
                    d="M13,0 L0,0"
                    id="Shape"
                  />
                  <path
                    d="M18,5 L0,5"
                    id="Shape"
                  />
                  <path
                    d="M13,10 L0,10"
                    id="Shape"
                  />
                </g>
              </g>
            </svg>
          </div>
        </div>
      </div>
      <div
        className="col-4 header_center"
      >
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
           
          <img
            alt=""
            src="logo.png"
          />
           
        </a>
      </div>
      <div
        className="col-4 header_right"
      >
        <ul>
          <li>
            <div
              className="search_c"
              onClick={[Function]}
            >
              <svg
                className="svg-icon searchicon"
                height="100%"
                style={Object {}}
                viewBox="0 0 32 32"
                width={35}
                xmlns="http://www.w3.org/2000/svg"
                xmlnsXlink="http://www.w3.org/1999/xlink"
              >
                <path
                  d="M23.5,24c-0.134,0-0.259-0.052-0.354-0.146l-9.004-9.004C12.582,16.238,10.588,17,8.5,17C3.813,17,0,13.187,0,8.5 C0,3.813,3.813,0,8.5,0C13.187,0,17,3.813,17,8.5c0,2.088-0.762,4.082-2.151,5.642l9.004,9.004C23.948,23.241,24,23.366,24,23.5 s-0.052,0.259-0.146,0.354S23.634,24,23.5,24z M8.5,1C4.365,1,1,4.365,1,8.5S4.365,16,8.5,16S16,12.635,16,8.5S12.635,1,8.5,1z"
                  fill="#000"
                />
              </svg>
            </div>
             
          </li>
          <li>
             
            <div
              className="cart_icon"
              onClick={[Function]}
            >
              <svg
                className="svg-icon "
                height="100%"
                style={Object {}}
                viewBox="0 0 32 32"
                width={35}
                xmlns="http://www.w3.org/2000/svg"
                xmlnsXlink="http://www.w3.org/1999/xlink"
              >
                <g
                  fill="none"
                  fill-rule="evenodd"
                  id="Page-1"
                  stroke="none"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  strokeWidth="1"
                >
                  <g
                    id="shopping-bag-skinny"
                    stroke="#000000"
                    transform="translate(1.000000, 1.000000)"
                  >
                    <rect
                      height="15"
                      id="Rectangle-path"
                      rx="1"
                      width="13"
                      x="0"
                      y="7"
                    />
                    <path
                      d="M3,5 L3,3.74999999 C3.00000003,1.67893217 4.5670034,-9.25185864e-17 6.5,0 C8.4329966,9.25185864e-17 9.99999997,1.67893217 10,3.74999999 L10,5"
                      id="Shape"
                    />
                  </g>
                </g>
              </svg>
            </div>
             
          </li>
        </ul>
      </div>
    </div>
  </div>
</header>
`;
